build/ci/unit-tests-template.yml

```diff
@@ -7,6 +8,13 @@ jobs:
     - script: $(Build.SourcesDirectory)\build.cmd /build /test /ci /sign /diagnostic /clearnugetcache /no-deploy /no-integration /no-ibc /configuration ${{ parameters.configuration }}
       displayName: Build ProjectSystem.sln
 
+    - ${{ if eq(parameters.enableRichCodeNavigation, true) }}:
+      - task: RichCodeNavIndexer@0
+        displayName: RichCodeNav Upload
+        inputs:
+          languages: 'csharp'
```

Comment:
We also have a lot of code on VB, is RichNav capable of that?

Reply:
Yes it is! With support from the Roslyn team, both C# and VB are ingested :)

Reply:
It's great to know your team utilizes VB. Once we get y'all set up with the clients, we'd love to hear more of your feedback on the VB experience :)